### PR TITLE
Update

### PR DESCRIPTION
**Pull request by jburel**
* Update the version of actions
* Test with updated test infra

### CODE REVIEW

**Comment by will-moore** [comment]
@jburel This PR should fix the `font.getsize()` issue reported by @juliomateoslangerak at https://github.com/ome/omero-figure/issues/522#issuecomment-1920999404

**Comment by jburel** [comment]
@will-moore I add to disable the ``test_move_annotations`` test for now. It failed for ``wells``.
I would suggest that we integrate the set of the changes in this PR after review then fix the test in a follow up PR since it is blocking other work

**Review by will-moore — APPROVED** [review]
Code changes look good. Tested Split-View Figure on merge-ci and it fixes the `font.getsize()` error and works OK. 👍

File: omero/util_scripts/Move_Annotations.py [hunk 1]
```diff
@@ -76,28 +76,28 @@ def get_key(ann_link, with_owner=False):
...
+    for link in old_links:
+        if get_key(link) in existing_well_keys:
             continue
-        log("    Annotation: %s %s" % (l.child.id.val,
-                                       l.child.__class__.__name__))
+        log("    Annotation: %s %s" % (link.child.id.val,
+                                       link.child.__class__.__name__))
         link = WellAnnotationLinkI()
         link.parent = WellI(well.id, False)
-        link.child = l.child
```
Comment (sbesson) [inline omero/util_scripts/Move_Annotations.py:97]: Isn't the test problem related to the fact that you renamed the loop variable as `link` but the logic also defines a different `link` objet?

This call is a no-op for instance

Comment (jburel) [reply omero/util_scripts/Move_Annotations.py:97]: well spotted. so many changes I missed it. I will check that

Comment (jburel) [reply omero/util_scripts/Move_Annotations.py:97]: test reactivated and everything is green

**Pull request merged**